src/styles/settings/_colors.css

```
@@ -0,0 +1,9 @@
+:root{
```

```suggestion
:root {
```

Pode deixar as variáveis todas num arquivo só, não precisa separar não. Pode ir tudo junto num settings.css ou algo assim.